security/monitoring.adoc

```diff
@@ -104,9 +104,14 @@ ifdef::openshift-enterprise[]
 - atomic-openshift-node
 endif::[]
 
-These logs are intended more for debugging purposes than for security auditing. You can retrieve logs for 
-each service with the `master-logs api api`, `master-logs controllers controllers`, or `master-logs etcd etcd` commands. 
-If your cluster runs an aggregated logging stack, like an xref:../install_config/aggregate_logging.adoc#aggregated-ops[Ops cluster], cluster administrators can retrieve logs from the logging _.operations_ indexes.
+These logs are intended more for debugging purposes than for security auditing. They can be
```

Comment:
:crying_cat_face: 
Please make these sentences active. I don't have it left in me.